src/main/java/com/wolox/training/models/Book.java

```diff
@@ -0,0 +1,112 @@
+package com.wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @Column(nullable = true)
```

Comment:
Hi @JoelIvanStella. If you're only going to define the non-nullability, don't use @column but instead use @NotNull (spring-boot-starter-validation). Change it for every case :+1: 

If variable is nullable, you can just leave it without annotations.

Also, leave a blank line between variables.

Reply:
The @column annotations are required to be able to generate the model in the database.
In this case I would add the @column (nulleable = false) annotation in conjunction with the @NotNull annotation(this should in the rest of the fields except for the id)

Reply:
Thanks guys! I pushed the change.